app/assets/stylesheets/home.scss

```diff
@@ -4,38 +4,131 @@ body {
   -moz-background-size: cover;
   -o-background-size: cover;
   background-size: cover;
+  height: 100%;
 }
 
-.footer {
+h1 {
+  color: white;
+  margin-top: 0;
+  padding: 15px 0 25px 0;
+}
+
+h3 {
+  color: white;
+}
+
+.logo img {
+  width: 40px;
+  margin: 3px 0 0 8px;
+}
+
+.t-logo img {
+  width: 55px;
+  padding-bottom: 5px;
+}
+
+.sign-in a {
+  text-align: center;
   background-color: #34495e;
 }
 
+#team {
+  background: image_url("success_team.png");
+  background-size: cover;
+  height: 150px;
+  width: 150px;
+}
+
+.site-footer {
+  background-color: #34495e;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.